commit cf262d7441d797942d33ed2a79540230fc5c97fa
Author: Christoph Langer <clanger@openjdk.org>
Date:   Thu Jan 12 08:02:58 2023 +0000

    8299439: java/text/Format/NumberFormat/CurrencyFormat.java fails for hr_HR
    
    Reviewed-by: naoto
    Backport-of: 3b374c0153950ab193f3a188b57d3404b4ce2fe2

diff --git a/src/jdk.localedata/share/classes/sun/util/resources/ext/CurrencyNames_hr_HR.properties b/src/jdk.localedata/share/classes/sun/util/resources/ext/CurrencyNames_hr_HR.properties
index 70f210e2da6..56e61953a8c 100644
--- a/src/jdk.localedata/share/classes/sun/util/resources/ext/CurrencyNames_hr_HR.properties
+++ b/src/jdk.localedata/share/classes/sun/util/resources/ext/CurrencyNames_hr_HR.properties
@@ -1,5 +1,5 @@
 # 
-# Copyright (c) 2005, 2012, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2005, 2023, Oracle and/or its affiliates. All rights reserved.
 # DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
 #
 # This code is free software; you can redistribute it and/or modify it
@@ -35,4 +35,5 @@
 # This notice and attribution to Taligent may not be removed.
 # Taligent is a registered trademark of Taligent, Inc.
 
+EUR=\u20AC
 HRK=Kn
diff --git a/test/jdk/ProblemList.txt b/test/jdk/ProblemList.txt
index dc167492272..a806ebe778a 100644
--- a/test/jdk/ProblemList.txt
+++ b/test/jdk/ProblemList.txt
@@ -1,6 +1,6 @@
 ###########################################################################
 #
-# Copyright (c) 2009, 2022, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2009, 2023, Oracle and/or its affiliates. All rights reserved.
 # DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
 #
 # This code is free software; you can redistribute it and/or modify it
diff --git a/test/jdk/java/text/Format/NumberFormat/CurrencySymbols.properties b/test/jdk/java/text/Format/NumberFormat/CurrencySymbols.properties
index cf73a9c7e31..665dd3b290a 100644
--- a/test/jdk/java/text/Format/NumberFormat/CurrencySymbols.properties
+++ b/test/jdk/java/text/Format/NumberFormat/CurrencySymbols.properties
@@ -79,7 +79,7 @@ fr_FR=\u20AC
 fr_LU=\u20AC
 hi_IN=\u0930\u0942
 hr=\u00A4
-hr_HR=Kn
+hr_HR=\u20AC
 hu=\u00A4
 hu_HU=Ft
 is=\u00A4
@@ -94,9 +94,9 @@ ja_JP=\uFFE5
 ko=\u00A4
 ko_KR=\uFFE6
 lt=\u00A4
-lt_LT=Lt;2014-12-31-22-00-00;\u20AC
+lt_LT=\u20AC
 lv=\u00A4
-lv_LV=Ls;2013-12-31-22-00-00;\u20AC
+lv_LV=\u20AC
 mk=\u00A4
 mk_MK=Den
 nl=\u00A4
